frontend/companion-app/Components.tsx

```diff
@@ -38,7 +38,7 @@ export function Score_Select({
           label: score.piece,
           value: score.filename,
         }))}
-        placeholder={{ label: "Select a score", value: null }}
+        placeholder={{ label: "Select a score", value: "air_on_the_g_string.musicxml" }}
```

Comment:
Can we remove this file now?

Reply:
Do you mean Components.tsx?  Or air_on_the_g_string.musicxml?  The former can be removed; App.tsx no longer depends on it.  The latter is referred to in the placeholder so that we can have a default score.

Reply:
I was talking about Components.tsx